deploy/base/easi.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: easi-backend-configmap
   namespace: easi
 data:
-  CEDAR_CORE_MOCK: true
-  CEDAR_INTAKE_ENABLED: false
+  CEDAR_CORE_MOCK: 'true'
+  CEDAR_INTAKE_ENABLED: 'false'
```

Comment:
curious - is this a styling preference? or do bools not work in this situation?

Reply:
Correct - unfortunately _currently_ ConfigMap values can only be strings. However _USUALLY_, once evaluated during container execution, the ENV variable will evaluate as a bool.

Goofy stuff.

Reply:
😵‍💫 😵‍💫 that is definitely weird but really cool to know.

Thanks!~